### DISCUSSION TITLE
Failed to create CoreCLR, HRESULT: 0x80070008

### DISCUSSION THREAD

**Discussion by Voidrunner42** [doitandbedone/ispyagentdvr-docker#518] [General]
**Affected Version**
Latest

**Describe the bug**
Fatal:  Exits.
Volume is, however utilized.

**Steps To Reproduce**
Steps to reproduce the behaviour:
1. Just run the container with the parameters from instructions.

**Reproduction rate**
100%

**Host System Information (where ispy agent is running/hosted):**
 - Bare-metal Alpine linux docker-host.  Latest build

**Comment by ispysoftware** [comment]
Googling Failed to create CoreCLR, HRESULT: 0x80070008...

https://www.linuxserver.io/blog/anatomy-of-an-issue

"Finally, one of the team reported being able to replicate the problem and we started digging into what was special about their setup. After a lot of back and forth and numerous test images being built we finally identified a difference: they were running a very out of date version of Docker. Specifically 19.03.5, which was released 2019-11-14. We know we've had issues in the past with older Docker versions causing problems so we got them to update to latest and retest: everything worked as expected."

**Comment by mitchray** [comment]
Encountered this on unRAID 6.10 which had Docker **v20.10.9**.

The issue is gone after updating unRAID to 6.11.1 with Docker **v20.10.18**

**Comment by marcolino7** [comment]
Hi,
encountered on Debian10 installed in vmWare 6.7 virtual machine.
Is there a way to fix? Thanks

**Reply by mitchray** [reply]
The fix appears to be updating docker, have you tried that?
